scripts/pre_push_hook.py

```diff
@@ -114,7 +116,10 @@ def get_remote_name():
     task = subprocess.Popen(get_remotes_name_cmd, stdout=subprocess.PIPE,
                             stderr=subprocess.PIPE)
     out, err = task.communicate()
-    remotes = python_utils.UNICODE(out)[:-1].split('\n')
+
+    # Get rid of Windows new line symbol.
+    out = python_utils.UNICODE(out)[:-1].replace('\r', '')
```

Comment:
Should only do this if platform is Windows, right?

(I know it doesn't really matter, but it seems conceptually more clean.)

Ditto elsewhere. Consider having a remove_windows_eol_characters() function somewhere if you need this to happen often.